internal/hostport/noop_hostport_manager.go

```diff
@@ -0,0 +1,21 @@
+package hostport
+
+import "github.com/sirupsen/logrus"
+
+type noOpHostportManager struct{}
```

Comment:
this is a matter of preference, feel free to ignore, but i'd prefer `noop` over `noOp` (same with `NewNoopHostportManager`)